src/OpenLoco/Windows/Construction/ConstructionTab.cpp

```diff
@@ -2472,26 +2475,124 @@ namespace OpenLoco::Ui::Windows::Construction::Construction
     }
 
     // 0x004A0AE5
-    void drawTrack(uint16_t x, uint16_t y, uint16_t selectedMods, uint16_t di, uint8_t trackType, uint8_t trackPieceId, uint16_t colour, uint8_t bh)
+    void drawTrack(const Map::Pos3& pos, uint16_t selectedMods, uint8_t trackType, uint8_t trackPieceId, uint8_t direction, Gfx::Context& context)
```

Comment:
Interestingly colour is not used. Perhaps this was leftover from RCT2 development. Instead the colour of the element is based on the company colour.